rattletrap.cabal

```
@@ -29,10 +29,10 @@ common basics
     bytestring ^>= 0.10.8,
     containers >= 0.5.11 && < 0.7,
     filepath ^>= 1.4.2,
-    http-client ^>= 0.5.13,
+    http-client >= 0.5 && < 0.7,
```

This change _might_ allow a version of `http-client` that doesn't actually work. Not super likely, but might as well keep the current version number. 
```suggestion
    http-client >= 0.5.13 && < 0.7,
```